oxalis-outbound/src/main/java/network/oxalis/outbound/transmission/DefaultTransmissionService.java

```diff
@@ -24,14 +24,14 @@
 
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
-import io.opentracing.Span;
-import io.opentracing.Tracer;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
 import network.oxalis.api.lang.OxalisContentException;
 import network.oxalis.api.lang.OxalisTransmissionException;
-import network.oxalis.api.tag.Tag;
 import network.oxalis.api.outbound.TransmissionResponse;
 import network.oxalis.api.outbound.TransmissionService;
 import network.oxalis.api.outbound.Transmitter;
+import network.oxalis.api.tag.Tag;
```

Comment:
Why unnecessary formatting (removing and adding import network.oxalis.api.tag.Tag) ?